examples/load_fw_canopen.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict
+
+from ingenialink import CAN_BAUDRATE, CAN_DEVICE
+from ingenialink.exceptions import ILFirmwareLoadError
+
+from ingeniamotion.motion_controller import MotionController
+
+
+def status_log(current_status: str) -> None:
+    print(f"Load firmware status: {current_status}")
+
+
+def progress_log(current_progress: int) -> None:
+    print(f"Load firmware progress: {current_progress}%")
+
+
+def load_firmware_canopen(can_drive: Dict[str, Any]) -> None:
+    # Create MotionController instance
+    mc = MotionController()
+    # Find available nodes
+    node_id_list = mc.communication.scan_servos_canopen(
+        can_drive["device"],
+        baudrate=can_drive["baudrate"],
+        channel=can_drive["channel"],
+    )
+    if can_drive["node_id"] not in node_id_list:
+        print("No nodes are connected.")
+        return
+    else:
+        print(f"Found nodes: {node_id_list}")
+
+    # Connect drive
+    print("Starts to established a communication.")
+    alias_can_servo = "custom_alias"
```

Comment:
Why do you use a custom alias? Could not be the default one for simplicity?

Reply:
It's easier without a custom alias, but we don't have a lot of examples using a custom alias. I think it's good having some to teach how to use them. What do you think?